rootfs/scheduler/__init__.py

```diff
@@ -89,6 +89,7 @@ def deploy(self, namespace, name, image, entrypoint, command, **kwargs):  # noqa
 
         app_type = kwargs.get('app_type')
         routable = kwargs.get('routable', False)
```

Comment:
correct, but I'm referring to other annotations such as IP whitelisting in https://github.com/deis/controller/issues/240.